README.md

```diff
@@ -5,22 +5,21 @@ This group works towards the development and adoption of ICRC standards related
-| [ICRC-27](./topics/icrc_27_get_icrc_1_accounts.md)                                                                | ICRC-1 ledger accounts<br>[![Extension Badge](https://img.shields.io/badge/Extends-ICRC--25-ffcc222.svg)](./topics/icrc_25_signer_interaction_standard.md)        | [sea-snake](https://github.com/sea-snake)                        | [![Status Badge](https://img.shields.io/badge/STATUS-DRAFT-ffcc00.svg)](https://github.com/orgs/dfinity/projects/31) | [Signer Standards Overview](https://github.com/dfinity/wg-identity-authentication/blob/main/topics/signer_standards_overview.md)           |
-| [ICRC-28](./topics/icrc_28_trusted_origins.md)                                                                    | Trusted Origins Standard                                                                                                                                          | [Dan Ostrovsky](https://github.com/dostro)                        | [![Status Badge](https://img.shields.io/badge/STATUS-DRAFT-ffcc00.svg)](https://github.com/orgs/dfinity/projects/31) |                                                                                                                                            |
-| [ICRC-29](./topics/icrc_29_window_post_message_transport.md)                                                      | Window Post Message Transport<br>[![Extension Badge](https://img.shields.io/badge/Extends-ICRC--25-ffcc222.svg)](./topics/icrc_25_signer_interaction_standard.md) | [Frederik Rothenberger](https://github.com/frederikrothenberger) | [![Status Badge](https://img.shields.io/badge/STATUS-DRAFT-ffcc00.svg)](https://github.com/orgs/dfinity/projects/31) | [Signer Standards Overview](https://github.com/dfinity/wg-identity-authentication/blob/main/topics/signer_standards_overview.md)           |
-| [ICRC-31](./topics/icrc_31_get_principals.md)                                                                     | Get Principals<br>[![Extension Badge](https://img.shields.io/badge/Extends-ICRC--25-ffcc222.svg)](./topics/icrc_25_signer_interaction_standard.md)                | [Philipp Litzenberger](https://github.com/plitzenberger)         | [![Status Badge](https://img.shields.io/badge/STATUS-DRAFT-ffcc00.svg)](https://github.com/orgs/dfinity/projects/31) | [Signer Standards Overview](https://github.com/dfinity/wg-identity-authentication/blob/main/topics/signer_standards_overview.md)           |
```

Comment:
Rather than removing the line completly, maybe it would be better to leave a note? e.g. "Superseded by ICRC-27"

Reply:
Added the line back and updated the status badge to indicate it's superseded by ICRC-27.

Reply:
Added the line back and updated the status badge to indicate it's superseded by ICRC-27.